fulcrumapp/fulcrum-core
# Feature:Add geometry to repeatables

# What?

- Adds geometry to repeatable values

# Why?

Enables viewing, creating, and modifying geometry for repeatables

# Testing
- In this PR: (to be added shortly!)
  - Create a repeatable section with advanced geometry enabled
  - See that you can create repeatable items with geometry
  - See the geometry in the repeatable view
